src/components/ProgressBar/index.tsx

```diff
@@ -1,64 +1,73 @@
 import React from 'react';
 import siluette from '../../assets/images/siluette.svg';
+type Props = {
+  sqSize: number;
+  shelters: number;
+  skids: number;
+  strokeWidth: number;
+  class: string;
+};
 
-export default class ProgressBar extends React.Component {
-  render() {
-    // Size of the enclosing square
-    const sqSize = this.props.sqSize;
-    // Enclose cicle in a circumscribing square
-    const viewBox = `0 0 ${sqSize} ${sqSize}`;
+function ProgressBar(props: Props) {
+  // Size of the enclosing square
+  const sqSize = props.sqSize;
+  // Enclose cicle in a circumscribing square
+  const viewBox = `0 0 ${sqSize} ${sqSize}`;
 
-    return (
-      <div className={`${this.props.class}`}>
-        <svg width={this.props.sqSize} height={this.props.sqSize} viewBox={viewBox}>
-          <image height="100px" width="100px" src={siluette} />
+  return (
+    <div className={`${props.class}`}>
+      <div style={{ maxWidth: '320px' }} className="relative xl:m-auto">
+        <img className="progressbar-image md:absolute" src={siluette} />
+        <svg width={props.sqSize} height={props.sqSize} viewBox={viewBox} className="xl:m-auto">
           <line
             className="circle-background"
             y1="260"
             y2="260"
             x1="0"
-            x2={this.props.copii}
-            transform={`rotate(-90 ${this.props.sqSize / 2} ${this.props.sqSize / 2})`}
-            strokeWidth={`${this.props.strokeWidth}px`}
+            x2={props.skids}
```

Comment:
skids? confused about naming here

Reply:
shelterkids era, ai vreo alta propunere, am vrut sa ramana si evident ce e ce

Reply:
ProgressBar ar trebui sa aiba props abstracte. nu putem folosi ceva generic ca sa putem folosi componenta si cua lte date?

Reply:
got it, on it